charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All changes to this chart will be documented in this file.
 * Update Chart's version to 10.7.0
 * Support Kubernetes v1.30
 * Upgrade ingress-nginx dependency to 4.10.1
+* Fix typo for Application Nodes `applicationNodes.podDisruptionBudget` instead of `searchNodes.podDisruptionBudget`
```

Comment:
Shall we remove it from the changelog? I find it too small to deserve a line there. Furthermore, if we keep it, we should mention in the chart manifest too. WDYT?

Reply:
fine by me, lets remove and just merge